### PR TITLE
nautilus: core: Health warnings on long network ping times

### PR DESCRIPTION
**Pull request by dzafman**
https://tracker.ceph.com/issues/41695

### CODE REVIEW

**Comment by dzafman** [comment]
We should add a backport of ceph/ceph#30220 here after it merges to master.

**Comment by dzafman** [comment]
Need to merge ceph/ceph#30220 first in case sha1 changes.

**Comment by dzafman** [comment]
jenkins test make check

File: src/osd/osd_types.cc [hunk 1]
```diff
@@ -431,14 +457,26 @@ void osd_stat_t::encode(bufferlist &bl, uint64_t features) const
   encode(num_shards_repaired, bl);
   encode(num_osds, bl);
   encode(num_per_pool_osds, bl);
```
Comment (liewegas) [inline src/osd/osd_types.cc]: need to bring in the num_per_pool_omap_osds field *before* this patch to avoid breaking encoding compatibility

Comment (dzafman) [reply src/osd/osd_types.cc]: @liewegas Switched to version 14 for compatibility, handling of version 13 for num_per_pool_omap_osds.

Comment (liewegas) [reply src/osd/osd_types.cc]: Perfect!

**Comment by yuriw** [comment]
[wip-yuri-testing-2019-11-05-2131-nautilus](https://trello.com/c/wV6UfGFb/747-wip-yuri-testing-2019-11-05-2131-nautilus)

**Pull request merged**